tasks/ubuntu.yml

```diff
@@ -105,11 +105,20 @@
   notify:
     - start zfs-share
 
+- name: ubuntu | Checking for ZFS-Auto-Snapshot package from the distribution
+  command: apt-cache show zfs-auto-snapshot
+  ignore_errors: true
+  register: auto_snapshot
+  when: zfs_enable_auto_snapshots
```

Comment:
As this is a read-only task you could add `changed_when: false` to prevent it from reporting changes.

You can also add `check_mode: false` for it to run when used with `--check`. Otherwise the following task fails because of missing `auto_snapshot.rc`

Reply:
@rp70 - Can you make these changes and we'll get this merged?

Reply:
> As this is a read-only task you could add `changed_when: false` to prevent it from reporting changes.
> 
> You can also add `check_mode: false` for it to run when used with `--check`. Otherwise the following task fails because of missing `auto_snapshot.rc`

Wow! It works like a charm. Thank you for the advice! I'm new to Ansible and still learning.

> @Rp70 - Can you make these changes and we'll get this merged?

Yes, I just made it. Let me know if you need anything else.

Thanks again!